src/main/kotlin/com/terraformation/backend/tracking/api/T0Controller.kt

```diff
@@ -49,6 +51,23 @@ class T0Controller(
     return GetAllSiteT0DataSetResponsePayload(allSet = allSet)
   }
 
+  @GetMapping("/plantingSite/{plantingSiteId}/species")
+  @Operation(
+      summary =
+          "Lists all the species that have been withdrawn to a planting site or recorded in " +
+              "observations (if not withdrawn).",
+      description =
+          "Species with densities are species that were withdrawn, species with null densities are " +
+              "species that were recorded in observations but not withdrawn to the plot's subzone.",
+  )
+  fun getSpeciesWithdrawnToPlantingSite(
```

Comment:
This method name seems inaccurate (and also, it'll collide with the existing endpoint's method name since OpenAPI endpoint names are a flat namespace; the OpenAPI schema generator will append `_1` to one of them.)